### PR TITLE
Add "The Mike Ratio"

### PR DESCRIPTION
**Pull request by MikeKlemarewski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -323,6 +323,35 @@ Try and leave this code a little better than you found it.
...
+one Mike.
+
+For millennia, Mikes (or Michaels) have been known to be highly efficient, top performers.
+Consider the following list of Michaels:
+
+- Michael Phelps
+- Michelangelo
+- Michael Jordan
+- Michael Caine
+- Mike D
```
Comment (kyoung) [inline README.md]: Jackson?

File: README.md [hunk 2]
```diff
@@ -323,6 +323,35 @@ Try and leave this code a little better than you found it.
...
+
+Looking at the Progressive Web team, we can see that having 3 Mike's has enabled the
+team to build a fantastic suite of tooling and components that will enable Mobify to
+build world class single page apps for our clients.
+
+We also need to consider diversity when we are hiring. Michelles are equally, yet
+sometimes more effective than Mikes in increasing productivity.
+
+There are no diminishing returns on the number of Mikes or Michelles. Ideally,
+your team would be %100 Mikes, Michaels and Michelles.
```
Comment (kyoung) [inline README.md]: `%` should come after the number.

File: README.md (hunk 1 above)
Comment (jeremywiebe) [inline README.md]: Michael Scott (The Office)?

Comment (kyoung) [inline README.md]: Also, Michael Palin, which supports the theory that great teams should have a Michael (ie. the Pythons)

File: README.md (hunk 2 above)
Comment (ericmuyser) [inline README.md]: Mike's. Always make you think.

Comment (MikeKlemarewski) [inline README.md]: Sorry, that's the Polish notation coming through

**Comment by jansepar** [comment]
Sorry Mike - closing this one down :)

**Pull request closed without merging**